metron-deployment/ansible/roles/librdkafka/defaults/main.yml

```diff
@@ -15,6 +15,6 @@
 #  limitations under the License.
 #
 ---
-librdkafka_version: 0.9.4
-librdkafka_url: https://github.com/edenhill/librdkafka/archive/v0.9.4.tar.gz
+librdkafka_version: 0.11.5
+librdkafka_url: https://github.com/edenhill/librdkafka/archive/v0.11.5.tar.gz
```

Comment:
Pycapa only works with 0.11.5 now.  This was tested and validated in a previous PR #1136.

Reply:
Should we update the bro package as well?  I would need to double check but I think it uses the version of librdkafka installed here in full-dev, but the package docs say "This plugin has been tested against the latest release of librdkafka, which at the time of this writing is v0.9.4."